data/card_catalogue/anger.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "title": "Positivity",
+    "upgrade": "Talk it Out",
+    "tier": "blue",
+    "details": {
+      "rules": "A visible target is forced to think positive thoughts",
+      "flavour": "",
```

Comment:
<code> \"You see a distant obstacle, I see an upcoming opportunity\" - Diggory the Myopic </code>